src/components/icons/NotAvailableSquareIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { ComponentWithAs } from '@chakra-ui/system';
+import { createIcon, IconProps } from '@chakra-ui/react';
+
+export const NotAvailableSquareIcon: ComponentWithAs<'svg', IconProps> =
```

Comment:
Do we want to control color by `currentColor` for this icon too?

Reply:
I changed the background color, but the dash color remains hardcoded.